ci-operator/config/open-cluster-management/observatorium-operator/open-cluster-management-observatorium-operator-release-2.1.yaml

```diff
@@ -0,0 +1,35 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.14
+images:
+- dockerfile_path: Dockerfile
+  to: observatorium-operator
+promotion:
+  name: observatorium-operator
+  namespace: open-cluster-management
+resources:
+  '*':
+    limits:
+      memory: 6Gi
+    requests:
+      cpu: 100m
+      memory: 1Gi
+tag_specification:
+  name: "4.4"
+  namespace: ocp
+test_binary_build_commands: "true"
+tests:
+- as: test
+  commands: |
+    KUBECTL=oc
+    OPERATOR_IMAGE_NAME=${IMAGE_FORMAT//\/stable:\$\{component\}//pipeline:observatorium-operator}
+    export KUBECTL OPERATOR_IMAGE_NAME
+    ./tests/e2e.sh deploy-operator
+  openshift_installer_src:
+    cluster_profile: aws
```

Comment:
`openshift_installer_src` is legacy and new jobs that use it should not be added. Multi-stage test wofkflows should be used instead. Please see https://docs.ci.openshift.org/docs/how-tos/migrating-template-jobs-to-multistage/#openshift_installer_src for information about how to write an equivalent in the newer mechanism, and https://docs.ci.openshift.org/docs/architecture/step-registry/ for  general information about multi-stage tests.